R/module_teal.R

```diff
@@ -22,6 +22,13 @@
 #' - data filtering in [`module_filter_data`]
 #' - data transformation in [`module_transform_data`]
 #'
+#' ## Fallback on failure
+#'
+#' `teal` is designed in such way that app will never crash if the error is introduced in any
+#' custom module provided by app developer (e.g. [teal_data_module()], [teal_transform_module()]).
```

Comment:
With the shiny module and teal module being referred as module it becomes a little ambiguous. And, I think `teal_data_module` and `teal_transform_module` are exhaustive. Example could indicate that this is a non-exhaustive.

```suggestion
#' custom shiny module provided by app developer ([teal_data_module()] or [teal_transform_module()])
```